dataproc/README.md

```diff
@@ -66,9 +66,11 @@ Note, we support specifying the hail_version, but only a very select number of v
 ### Rebuilding the dataproc driver image
 
 ```sh
+# from repo root
+cd dataproc
 gcloud config set project analysis-runner
 # grab the HAIL_VERSION from analysis_runner/dataproc.py
-HAIL_VERSION=$(grep "DEFAULT_HAIL_VERSION = '" analysis_runner/dataproc.py | awk -F\' '{print $2}')
+HAIL_VERSION=$(grep "DEFAULT_HAIL_VERSION = '" ../analysis_runner/dataproc.py | awk -F\' '{print $2}')
```

Comment:
could shuffle this line to before `cd dataproc` and skip the `../`

Reply:
Aye aye, good shout